### DISCUSSION TITLE
Copy Talkgroups from one "system" to another

### DISCUSSION THREAD

**Discussion by dthomson134** [chuot/rdio-scanner#415] [Q&A]
Background: I have a TrunkRecorder system monitoring three simulcast systems on a statewide system.

1) I tried to import a talkgroup list from RadioReference, but I was unable to import them to an existing system, it created a new "system."
    Is there a way to import them to an existing system?

2) I manually updated hundreds of talkgroups on one of the "systems" and I want to share/copy those talkgroups to another "system."  
    Is there a way to copy those, either through the interface, or a database copy?  
    Is there a way to access the internal DB for rdio-scanner?

Thanks much!

**Comment by Thinline-Dynamic-Solutions** [comment]
Not to my knowledge. I just got done doing all of mine this way.

*Michael Chambers - Ohio Radio Scanner Network*
Founder/Manager/Support
***@***.**** ***@***.***>

**Comment by tadscottsmith** [comment]
From the Admin GUI you can export the config to JSON. You might be able to copy and modify portions of that JSON and then Import it into the other system. I'd make sure you have a good backup first.
